junifer/api/tests/test_functions.py

```diff
@@ -240,32 +247,37 @@ def test_queue_correct_yaml_config(
         The path to the test directory.
     monkeypatch : pytest.MonkeyPatch
         The monkeypatch object.
```

Comment:
Should you use CamelCase in the docstring here, similar to the DataGrabber issue?

Reply:
Resolving this as I'll take care of this in a separate PR.